en/docs/guides/identity-lifecycles/extend-username-and-password-recovery.md

```diff
@@ -0,0 +1,534 @@
+
+    Note: You can skip this step if there are no entries for `event.default_listener.xxx` in the `deployment.toml` file.
+    
+    ``` java
+    [event.default_listener.identity_mgt]
+    priority= "50"
+    enable = false
+    [event.default_listener.governance_identity_mgt]
+    priority= "95"
+    enable = true
+    [event.default_listener.governance_identity_store]
+    priority= "97"
+    enable = true
+    ```
+
+## Set up password recovery
+
+Follow the steps given below to recover a user in the super tenant (i.e., `carbon.super`).
+
+1. Add the following properties to the `deployment.toml` file in the `IS_HOME/repository/conf` folder to configure the WSO2 Identity Server to send confirmation emails.
+
+    ``` toml
+    [output_adapter.email]
+    from_address= "wso2iamtest@gmail.com"
+    username= "wso2iamtest"
+    password= "Wso2@iam70"
+    hostname= "smtp.gmail.com"
+    port= 587
+    enable_start_tls= true
+    enable_authentication= true
+    ```
+
+    !!! Note
+        - Add this configuration only if you wish to configure WSO2 Identity Server to send confirmation emails. See [Configure the Email Sending Module]({{base_path}}/deploy/configure-email-sending/) for more information
+        - Alternatively, you can use your own email managing mechanism.
+
+
+2. Add the following properties to the `deployment.toml`.
+
+    ``` toml
+    [identity_mgt.notification_channel_recovery]
+    recovery_code_validity=2
+    resend_code_validity=5
+    
+    [identity_mgt.password_reset_sms]
+    sms_otp_validity=2
+    ```
+
+    <table>
+       <tr>
+           <th>Property Name</th>
+           <th>Functionality</th>
+       </tr>
+       <tr>
+          <td><code>recovery_code_validity</code></td>
+          <td>
+            <ul>
+                <li>Validity period of the recovery code given after initiating username/ password
+                recovery.
+                <li>Unit : <code>minutes</code>
+                <li>Default value : <code>1 minute</code>
+            </ul>
+            <div class="admonition note">
+                <p class="admonition-title">note</p>
+                <p>
+                    If you have configured Username Recovery Via User Preferred Notification Channel, the above field will already be configured.
+                </p>
+            </div>
+          </td>
+       </tr>
+       <tr>
+          <td><code>resend_code_validity</code></td>
+          <td>
+            <ul>
+                <li>Validity period of the recovery code given after initiating password recovery.
+                <li>Unit : <code>minutes</code>
+                <li>Default value : <code>1 minute</code>
+            </ul>
+          </td>
+       </tr>
+       <tr>
+         <td><code>sms_otp_validity</code></td>
+         <td>
+           <ul>
+                <li>Validity period of SMS OTP when the selected channel is <code>SMS</code>.
```

Comment:
same comment as before.. 
Please correct me if I am wrong. Don't see how this is a list.